sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java

```diff
@@ -70,6 +70,31 @@ public CreateNewCaseSteps(WebDriverHelpers webDriverHelpers, CaseService caseSer
           webDriverHelpers.clickOnWebElementBySelector(CASE_SAVED_POPUP);
         });
 
+      When(
+              "^I fill new case form with specific data$",
```

Comment:
CreateNewCaseSteps -> ^I create a new case with specific data$
This one is already created here, most of the create/edit entities are covered in framework, excepting Immunizations which are untouched :D